validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -82,7 +82,7 @@ public abstract class SequenceBase {
   public static final String EVENT_PREFIX = "event_";
   public static final String SYSTEM_EVENT_MESSAGE_BASE = "event_system";
   public static final int CONFIG_UPDATE_DELAY_MS = 2000;
-  public static final int NORM_TIMEOUT_MS = 120 * 1000;
+  public static final int NORM_TIMEOUT_MS = 300 * 1000;
```

Comment:
Is this change still necessary?  I'm worried about this because it starts to drastically increase the overall time.  Is this because the new tests that you added have longer times to them but there's no way to individually specify an increased timeout?

Reply:
This test needed a longer time to run, but the global timeout overrides the test level timeout. So I increased the overall time, then put a timeout on every test to keep the time low

Reply:
Well, I hate this logic but it's not your fault! It should be the other way around... default is shorter and then if you want longer it's a special case!

Reply:
(Nothing needs to change here, I'm just ranting)